frontend/src/app/signUp/Organization/utils.tsx

```diff
@@ -147,6 +147,15 @@ export const organizationSchema: yup.SchemaOf<OrganizationCreateRequest> = yup
 
 export const organizationBackendErrors = (error: string): ReactElement => {
   switch (error) {
+    // Unusable org name.  Name probably doesn't contain any alphanumeric characters
+    case "The organization name is empty.":
+    case "The organization name is invalid.":
+      return (
+        <Alert type="error" title="Invalid organization name">
+          The organization name you entered is invalid. Please double check and
+          re-enter the organization name.
```

Comment:
@Rebecca-LM Does this error message look alright to you? It's for the toast users will see if they enter an invalid name (one that's empty or only consists of special characters, for example)

Reply:
Yes LGTM! I sent this copy to Brett :)